### ISSUE TITLE
Model Training Notebook(dropout layers)

### ISSUE THREAD

**Issue by Enrique2007s** [Enrique2007s/skin-lesion-detector#15]
As a data scientist, I want to use dropout layers to prevent overfitting of the model.

Action: 
Define neural network architecture with dropout layers
